components/inputs/test/input-text.test.js

```diff
@@ -35,41 +35,6 @@ function pressEnter(elem) {
 
 describe('d2l-input-text', () => {
 
```

Comment:
Were these changes intended to be in here?

Reply:
Yup! I was overall trying to improve the build and these axe tests are also in a `.axe` file and were occasionally failing on edge. The change to `localize-mixin` was similar - that test was flaky as well (on other browsers too) for the `async` case and after bumping the timeout I have not seen it fail again.